core/jobs.py

```diff
@@ -222,11 +220,9 @@ def _compress_output_list(cls, output_list, test_only_max_output_len=None):
         ]
```

Comment:
Can we mention (perhaps in the name of the constant, or the docstring) the units? Maybe add _CHARS at the end.